experimental/MatrixGroups/src/matrix.g

```diff
@@ -123,14 +123,14 @@ InstallMethod(SetNiceMorphismForJuliaMatrixRepGroup, [IsGroup], function(G)
         Assert(0, ForAll(gens, IsJuliaMatrixRep));
         
         ele := gens[1];
-        hom := Julia.Oscar.iso_oscar_gap(Julia.base_ring(ele!.m));
+        hom := Oscar_jl.iso_oscar_gap(Julia.base_ring(ele!.m));
```

Comment:
We get warnings about `Oscar_jl` being unbound here. I think this might be caused by this code being loaded before Oscar.jl ?

The easiest way to fix this would be to move this file into `OscarInterface`.

Reply:
(and then edit `experimental/MatrixGroups/src/MatrixGroups.jl` to remove its `__init__` method)

Reply:
Like this: https://github.com/oscar-system/Oscar.jl/pull/4577/commits/7f589585455272e9d1997f23a6000caad14b7f43?